### DISCUSSION TITLE
How to bind Comand to every ItemsRepeater items with item index or itself as argument

### DISCUSSION THREAD

**Discussion by hheimerd** [AvaloniaUI/Avalonia#7856] [Q&A]
Hi! I need some help 🙏
I have a list with models. This is shown using an ItemsRepeater using Items prop. The element template contains a button. 
I want that when the button is clicked, the corresponding page of the model opens (as a link). How can I do something like this?

It would be nice if it matched the mvvm pattern

**Comment by wieslawsoltes** [comment]
You can use https://github.com/wieslawsoltes/AvaloniaBehaviors and apply behavior using style to all items repeater children.
